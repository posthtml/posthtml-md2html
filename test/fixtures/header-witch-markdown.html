<markdown># Heading with *italics*</markdown>
<markdown>**Bold** paragraph</markdown>
<div markdown>

  | Head | row |
  |------|-----|
  | Data | row |

</div>
<main markdown>
> # Title
>
> This is blockquote paragraph
</main>
