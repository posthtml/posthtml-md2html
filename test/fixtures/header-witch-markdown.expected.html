<h1 id="heading-with-italics">Heading with <em>italics</em></h1>

<p><strong>Bold</strong> paragraph</p>

<div><table>
<thead>
<tr>
<th>Head</th>
<th>row</th>
</tr>
</thead>
<tbody><tr>
<td>Data</td>
<td>row</td>
</tr>
</tbody></table>
</div>
<main><blockquote>
<h1 id="title">Title</h1>
<p>This is blockquote paragraph</p>
</blockquote>
</main>
